<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>exeptionMessage</title>
</head>
<body>
    
    <script>
    const test = () => {
        try {
            const array = new Array(999999999999999999); //예외 발생
        } catch (exception) {
            console.log(exception); //RangeError 는 어떤 값이 집합에 없거나, 허용되는 범위가 아닐 때 나타나는 오류
            console.log();
            console.log(`예외 이름: ${exception.name}`);
            console.log(`예외 메시지: ${exception.message}`);
        }
        
    }

        test();
       
    </script>
</body>
</html>